gradient/cli/notebooks.py

```diff
@@ -106,6 +106,12 @@ def notebook_metrics():
     type=bool,
     cls=common.GradientOption,
 )
+@click.option(
+    "--environment",
+    "environment",
+    type=json_string,
+    help="Environmental variables",
```

Comment:
The "environment" variable is not loaded from yaml file because you have to use
cls=common.GradientOption,
here. Loading values happens in the GradientOption method.Eevery option has to have the cls set to GradientOption for the yaml loading to work. It's not s perfect solution as one have to remember to put it everywhere but I couldn't find any better at the time I was working on it.

Reply:
Ah I must have lost that in the merge. Thanks! I thought I was going nuts.